iac-extensions/spring-config/src/test/java/org/sonar/iac/springconfig/parser/properties/PropertiesParserBaseVisitorTest.java

```diff
@@ -41,7 +43,7 @@ void shouldParseSimpleKeyValue() {
     parseProperties(code);
 
     assertThat(visitor.visited()).containsExactly(
-      "visitPropertiesFile foo=bar<EOF>",
+      "visitPropertiesFile foo=bar<EOF><EOF>",
```

Comment:
Why the double EOF?

Reply:
I changed the following rule:

```
propertiesFile
    : row* EOF
    ;
```

I added EOF here.

Thanks to that the whole content of the input will be consumed and I was able to raise error on `=abc` input. Before that such input produced empty `PropertiesFile`.

My guess is that the double `<EOF>` is not a problem at all.